lib/checker_jobs/notifiers/logger.rb

```diff
@@ -0,0 +1,53 @@
+require 'logger'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.